core/src/commonMain/kotlin/com/sourcepoint/mobile_core/Coordinator.kt

```diff
@@ -84,23 +83,27 @@ class Coordinator(
             usnat = campaigns.usnat?.let { SPUserData.SPConsent(consents = state.usNat.consents) }
         )
 
+    @Suppress("Unused")
```

Comment:
This is helpful! I didn't know about that.

Reply:
I try to use it sparingly, because in the end I want the IDE to tell us if a code is unused for real. But if know the code is used in one of the SDKs for sure, this is helpful.